guideon-core/src/main/resources/application.yml

```
@@ -0,0 +1,37 @@
+server:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)